src/vs/vscode.d.ts

```diff
@@ -4790,6 +4790,11 @@ declare module 'vscode' {
 		 * Controls whether the terminal is cleared before executing the task.
 		 */
 		clear?: boolean;
+
+		/**
+		 * Controls whether the task is executed in a specific terminal group using split panes.
+		 */
+		terminalGroup?: string;
```

Comment:
This will need to go into vscode.proposed.d.ts. Once the API change has gone through our review process I'll move it to vscode.d.ts.

Reply:
I made a comment on the request about this https://github.com/Microsoft/vscode/issues/47265#issuecomment-452034696, I'd like to be in the API call when it happens (I should be in the next one).

Reply:
I didn't even realize I was changing the API :smile: Moved it into `vscode.proposed.d.ts` as requested.

Reply:
@Tyriar, sounds good. I'll bring it up for the next API call.
@cmfcmf, expect some more change requests sometime next week from the API discussion. I think you're heading in a good direction here though, and I don't expect any big function changes.